apps/sertra/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sertra",
```

Comment:
To keep naming consistent with other apps, this should probably be `@thunderstore/sertra`?